### PR TITLE
[kibana] - Add additionals labels

### PR DESCRIPTION
**Pull request by plumcraft**
- [x] Chart version *not* bumped (the versions are all bumped and released at the same time)
- [x] README.md updated with any new values or changes
- [x] Updated template tests in `${CHART}/tests/*.py` 
- [x] Updated integration tests in `${CHART}/examples/*/test/goss.yaml`

### CODE REVIEW

**Comment by plumcraft** [comment]
Hello, i'm not sure if i need to do something on my side ? 

Thanks

**Comment by Crazybus** [comment]
@plumcraft The [message from elasticcla](https://github.com/elastic/helm-charts/pull/225#issuecomment-512754219) describes what you need to do. Once you have added the email address you used in the commit just leave a new comment and it will re-run the cla check. 
 
jenkins test this please

**Comment by plumcraft** [comment]
@Crazybus : thanks for you feedback, i've added my another email into my github profile, it should be fine now.

**Comment by Crazybus** [comment]
jenkins test this please

File: kibana/tests/kibana_test.py [hunk 1]
```diff
@@ -355,4 +355,12 @@ def test_override_imagePullPolicy():
 
     r = helm_template(config)
     c = r['deployment'][name]['spec']['template']['spec']['containers'][0]
-    assert c['imagePullPolicy'] == 'Always'
\ No newline at end of file
+    assert c['imagePullPolicy'] == 'Always'
+
+  def test_adding_pod_labels():
```
Comment (Crazybus) [inline kibana/tests/kibana_test.py]: This indentation here is causing the test to fail:

```
17:57:06 ==================================== ERRORS ====================================
17:57:06 ____________________ ERROR collecting tests/kibana_test.py _____________________
17:57:06 /usr/local/lib/python2.7/site-packages/_pytest/python.py:445: in _importtestmodule
17:57:06     mod = self.fspath.pyimport(ensuresyspath=importmode)
17:57:06 /usr/local/lib/python2.7/site-packages/py/_path/local.py:701: in pyimport
17:57:06     __import__(modname)
17:57:06 E     File "/app/kibana/tests/kibana_test.py", line 360
17:57:06 E       def test_adding_pod_labels():
17:57:06 E                                   ^
17:57:06 E   IndentationError: unindent does not match any outer indentation level
17:57:06 !!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
17:57:06 =========================== 1 error in 0.24 seconds ============================
```

```suggestion
def test_adding_pod_labels():
```

**Comment by Crazybus** [comment]
jenkins test this please

**Comment by Crazybus** [comment]
Thank you for adding this in! Extra bonus points for adding a test too :)

**Pull request merged**